lib/chat_api/widget_configs.ex

```diff
@@ -0,0 +1,118 @@
+defmodule ChatApi.WidgetConfigs do
+  @moduledoc """
+  The WidgetConfigs context.
+  """
+
+  import Ecto.Query, warn: false
+  alias ChatApi.Repo
+
+  alias ChatApi.WidgetConfigs.WidgetConfig
+
+  @doc """
+  Returns the list of widget_configs.
+
+  ## Examples
+
+      iex> list_widget_configs()
+      [%WidgetConfig{}, ...]
+
+  """
+  def list_widget_configs do
+    Repo.all(WidgetConfig)
+  end
+
+  @doc """
+  Gets a single widget_config.
+
+  Raises `Ecto.NoResultsError` if the Widget config does not exist.
+
+  ## Examples
+
+      iex> get_widget_config!(123)
+      %WidgetConfig{}
+
+      iex> get_widget_config!(456)
+      ** (Ecto.NoResultsError)
+
+  """
+  def get_widget_config!(id), do: Repo.get!(WidgetConfig, id)
+
+  def create_or_update(nil, params) do
+      create_widget_config(params)
```

Comment:
looks like the indentation is a tiny bit off -- mind running `mix format` just to double check all the formatting?